src/main/java/seedu/address/logic/commands/FindTagCommand.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.commands;
+
+import seedu.address.model.person.TagsContainKeywordsPredicate;
+
+/**
+ */
+public class FindTagCommand extends Command {
+
+    public static final String COMMAND_WORD = "findtag";
+    public static final String COMMAND_ALIAS = "fitg";
```

Comment:
support for COMMAND_ALIAS has been removed now, fyi
you can remove the constant from the class.